### PR TITLE
[MRG] rename get_values_for_key -> get_entity_vals

### PR DESCRIPTION
**Pull request by sappelhoff**
Perhaps this is a good compromise @jasmainak @teonbrooks 

- [x] less verbose
- [x] less generic
- [x] is not a misnomer

Merge checklist
---------------

Maintainer, please confirm the following before merging:

- [x] All comments resolved
- [ ] This is not your own PR
- [x] All CIs are happy
- [x] PR title starts with [MRG]
- ~~[whats_new.rst](https://github.com/mne-tools/mne-bids/blob/master/doc/whats_new.rst) is updated~~
- ~~PR description includes phrase "closes <#issue-number>"~~
- [x] Commit history does not contain any merge commits

### CODE REVIEW

File: mne_bids/utils.py [hunk 1]
```diff
@@ -55,38 +55,40 @@ def get_kinds(bids_root):
...
+    key.
 
     Parameters
     ----------
     bids_root : str
         Path to the root of the BIDS directory.
-    key : str
-        The name of the key to search for. Can be one of
+    entity_key : str
+        The name of the entity key to search for. Can be one of
```
Comment (jasmainak) [inline mne_bids/utils.py]: Can you add a tiny doc example below to show what it will return?

Comment (sappelhoff) [reply mne_bids/utils.py]: done in https://github.com/mne-tools/mne-bids/pull/260/commits/0a4ba164a25163791e8612a7191f1a8688b7f5ae

**Comment by jasmainak** [comment]
Thanks @sappelhoff !

**Comment by jasmainak** [comment]
can you update this in mne-study-template? :) don't forget about it ;-)

**Pull request merged**